ext/openssl/ossl_pkey_rsa.c

```diff
@@ -626,6 +626,174 @@ ossl_rsa_to_public_key(VALUE self)
     return obj;
 }
 
+#if (OPENSSL_VERSION_NUMBER >= 0x1000100f)
+/*
+ * call-seq:
+ *    rsa.sign_pss(digest, data, salt_len, hash_alg) -> String
+ *
+ * Probabilistic Signature Scheme for RSA sign().
+ *
+ * To sign the +String+ +data+, +digest+ must be provided.
+ * +digest+ must be a OpenSSL::Digest instance or +String+ like "SHA1".
+ *
+ * The +Integer+ +salt_len+ should be the salt length to use.
+ *
+ * The +String+ +hash_alg+ should be the hash algorithm used in MGF.
```

Comment:
Please make it clear that this is a parameter of MGF1, which is just one implementation of MGF (mask generation function).